main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunner.java

```diff
@@ -159,6 +164,33 @@ public static void printSummary(
           skippedValidators.stream().map(Class::getSimpleName).collect(Collectors.joining(",")));
       logger.atSevere().log(b.toString());
     }
+
+    List<Class<? extends FileValidator>> validatorsWithParsingErrors =
```

Comment:
Shouldn't the 3 "skipped" lists (skippedValidators, validatorsWithParsingErrors, singleEntityValidatorsWithParsingErrors) all be listed together?
I thought all the validators in these lists were not run because of dependency problems.

Reply:
Changes are pushed.